tests/imputers/test_ArbitraryImputer.py

```diff
@@ -167,3 +77,15 @@ def test_impute_value_preserve_dtype(self):
         # Checking if the dtype of "a" and "b" are int8 and float16 respectively after imputation
         assert df["a"].dtype == "int8"
         assert df["b"].dtype == "float16"
+
+
```

Comment:
Why are we not using GenericInitTests here?

Reply:
as in why ColumnStrListInitTests?

ColumnStrListInitTests inherits from GenericInitTests and adds checks for columns=List[str], so just adds a few more relevant tests to GenericInitTests :)